tsl/test/sql/include/skip_scan_dagg_load_comp_query.sql

```diff
@@ -48,9 +48,11 @@ SET max_parallel_workers_per_gather = 0;
 :PREFIX SELECT count(DISTINCT dev), dev FROM :TABLE GROUP BY dev ORDER BY dev DESC LIMIT 3;
 
 \qecho range queries on :TABLE
+set enable_seqscan to off;
```

Comment:
It's a toy table with two compressed batches with ranges about [0, 250], so there's no way we'd get predictable estimates for that. Had to disable seq scan.